packages/nx-plugin/src/generators/playwright-component-configuration/files/playwright.config.ts.template

```diff
@@ -33,7 +33,8 @@ export default defineConfig({
     trace: 'on-first-retry',
 
     /* Port to use for Playwright component endpoint. */
-    ctPort: 3100,
+    // eslint-disable-next-line @typescript-eslint/no-var-requires
+    ctPort: 3100 + Number(require('./package.json').name.length),
```

Comment:
this is only way to avoid port clashes when running multiple ct test both localy and on CI. 

once we migrate to nx inferred projects this will be generated behind the scenes